### PR TITLE
Update to 0.19

### PR DESCRIPTION
**Pull request by michaeljones**
Via elm-upgrade. The elm-format changes have not been commited. I don't know if we want the 'bugfix2' in the package.json yet or if there is a better way to approach it.

I hope this is useful but it if you already had plans for how to go about this then feel free to close.

Thanks!

### CODE REVIEW

**Comment by joneshf** [comment]
Thanks very much. Will review this weekend!

File: elm-package.json [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{
```
Comment (joneshf) [inline elm-package.json]: I think I'd like to hold onto the 0.18 and older manifest for a while longer, at least for a couple of version releases. I'd hate to have to force people to upgrade to 0.19 in order to get a bug fix or a feature.

Comment (michaeljones) [reply elm-package.json]: Sure, makes sense. I can amend the commit.

**Comment by michaeljones** [comment]
Amended :)

**Comment by joneshf** [comment]
I see what's going on with Travis. It's using an out of date version of node.

Since there's bound to be some tedious changes to make Travis work again, I won't force the burden on you. It will be a day or two until I can get to it, though. I don't want to stop you from fixing it if you're blocked on this, either.

**Pull request merged**